### PR TITLE
[RFC] Use the BatchInserter for faster loading of graphs, and remove the Neo4jSingleton.

### PR DESCRIPTION
**Pull request by Balletie**
Comes after #98.

### CODE REVIEW

File: src/main/java/nl/tudelft/dnainator/ui/controllers/FileOpenController.java [hunk 1]
```diff
@@ -139,6 +147,11 @@ private void onOpenAction() {
 		animation.toggle();
 
 		if (graphLoadService.getNodeFile() != null && graphLoadService.getEdgeFile() != null) {
+			try {
+				FileUtils.deleteRecursively(new File(graphLoadService.getDatabase()));
+			} catch (Exception e) {
+				e.printStackTrace();
```
Comment (Unia) [inline src/main/java/nl/tudelft/dnainator/ui/controllers/FileOpenController.java]: Perhaps we should show an `ExceptionDialog`?

**Comment by Unia** [comment]
Next reviewer may merge after @gfokkema has added a TODO at FileOpenController:149

**Comment by gfokkema** [comment]
See #118 for welcome screen requirements.
These should supersede the deletion of the db in FileOpenController.

File: src/main/java/nl/tudelft/dnainator/ui/services/GraphLoadService.java [hunk 2]
```diff
@@ -95,22 +106,18 @@ public final String getDatabase() {
...
 				EdgeParser ep = new EdgeParserImpl(getEdgeFile());
 				NodeParser np = new NodeParserImpl(getNodeFile());
-
 				gb.constructGraph(np, ep);
 
 				ep.close();
 				np.close();
 
-				return gb;
+				Graph graph = new Neo4jGraph(database.get());
```
Comment (pvanagtmaal) [inline src/main/java/nl/tudelft/dnainator/ui/services/GraphLoadService.java]: No need to create this variable

**Pull request merged**